docs/samples/tensorflow/README.md

```diff
@@ -85,7 +85,7 @@ NAME             READY     URL                                  DEFAULT TRAFFIC
 flowers-sample   True      http://flowers-sample.default.example.com   90                10               48s
 ```
 
-If you are using the [Knative CLI (knctl)](#knative-cli), run the following command
+If you are using the [Knative CLI (knctl)](https://github.com/cppforlife/knctl), run the following command
```

Comment:
I think `knctl` is no longer in active development, we should switch to https://github.com/knative/client

Reply:
@yuzisun Dan, I assume you are recommending to use kn instead of  knctl.  If we use kn, the command and output need to change to the following:
kn route list
NAME                               URL                                                           AGE     CONDITIONS   TRAFFIC
flowers-sample-predictor-canary    http://flowers-sample-predictor-canary.default.example.com    11m     3 OK / 3     100% -> flowers-sample-predictor-canary-mswmr
flowers-sample-predictor-default   http://flowers-sample-predictor-default.default.example.com   2d20h   3 OK / 3     100% -> flowers-sample-predictor-default-x7zcl
-------------------------------------------------------------------------------------------------
Is this OK?

Reply:
@yuzisun Dan, I assume you are recommending to use kn instead of knctl. If we use kn, the command and output need to be changed to the following. What do you think?

kn route list
NAME URL AGE CONDITIONS TRAFFIC
flowers-sample-predictor-canary http://flowers-sample-predictor-canary.default.example.com 11m 3 OK / 3 100% -> flowers-sample-predictor-canary-mswmr
flowers-sample-predictor-default http://flowers-sample-predictor-default.default.example.com 2d20h 3 OK / 3 100% -> flowers-sample-predictor-default-x7zc

Reply:
ye this looks good to me! Thanks!

Reply:
Thanks Dan! Will change.